crates/bevy_android/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "bevy_android"
+version = "0.9.0"
+edition = "2021"
+description = "Provides handle to AndroidApp for Bevy Engine"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+keywords = ["bevy"]
+
+[dependencies]
+bevy_app = { path = "../bevy_app", version = "0.9.0" }
+bevy_ecs = { path = "../bevy_ecs", version = "0.9.0" }
+bevy_log = { path = "../bevy_log", version = "0.9.0" }
+once_cell = "1.16"
+
+ndk = "0.7" # This version *must* be the same as the version used by android-activity
```

Comment:
I'm not sure this is true, but might be overlooking something.

The ndk crate shouldn't have any static state, though there are some ndk types exposed by the android-activity crate, e.g. for input.

I would have expected there wouldn't be any conflict with using other versions, but maybe it's an issue considering that the only use of the `ndk` crate in Bevy is in conjunction with an `android-activity` API and some types don't match without picking the same version here.

Maybe android-activity and Winit could re-export the `ndk` API similar to how the Winit crate re-exports the android-activity crate. That way you wouldn't have to worry about synchronizing this if you're only interested in using the same version that Winit and android-activity use.

This isn't the same as the previous requirement for the `ndk-glue` crate version needing to match the version used by Winit. There is a similar requirement that the `android-activity` crate must match watch Winit expects (for the same reason todo with how these glue crates are responsible for the main entrypoint of the application)